src/sonic-config-engine/tests/sample_output/py2/ports.json

```diff
@@ -20,18 +20,207 @@
         },
         "OP": "SET"
     },
+    {
+        "PORT_TABLE:Ethernet108": {
+            "speed": "10000",
+            "description": "fortyGigE0/108"
```

Comment:
What will fail if you do not add?

Reply:
https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-config-engine/tests/test_j2files.py#L55-L59
This case will fail